app/assets/stylesheets/modules/_compliments.scss

```diff
@@ -0,0 +1,98 @@
+.send {
+  border: 1px solid $white;
+  float: right;
```

Comment:
Properties should be ordered border, border-radius, float, padding, -moz-transition, -o-transition, -webkit-transition, transition